tempest/api/workloadmgr/security_groups/test_security_group_remotesg_cyclic.py

```diff
@@ -743,7 +743,7 @@ def test_with_cyclicsg_03(self):
                 # Delete restored vms and security groups created during earlier restore
                 if test != tests[-1]:
                     LOG.debug("deleting restored vm and restored security groups and rules")
-                    self._delete_vms_secgroups(restored_vms, secgrp_val)
+                    self._delete_vms_secgroups(restored_vms, self.restored_secgroup_ids)
```

Comment:
One question - Under PR#370 and PR#371 in test_security_group_remotesg_cyclic.py at line number #746 the code is bit different. 
In 4.2 you are passing secgrp_val and in 5.0 you are passing self.restored_secgroup_ids

Any specific reason ?

self._delete_vms_secgroups(restored_vms, secgrp_val)
self._delete_vms_secgroups(restored_vms, self.restored_secgroup_ids)

Reply:
updated in 4.2 as well